onboard/catkin_ws/src/avt_camera/calibrations/DEV_000F315C1ED5.yaml

```diff
@@ -0,0 +1,18 @@
+!!python/unicode 'camera_matrix':
```

Comment:
How were these generated? Do they work?

Reply:
They do work. I believe they were generated using the camera calibration package in ROS.